viz.Rmd

```diff
@@ -529,11 +770,11 @@ morley_hist <- ggplot(morley, aes(x = Speed, fill = Expt)) +
 morley_hist
```

Comment:
do we explain `position = "identity"` -- I think we started using it here, but don't think it's explained... I think we should remove it here since it doesn't do anything in this particular line. 
But we should explain it below when the colours are introduced below

Reply:
We explain it in the landmass example. We don't even really need it here at all... I don't think...

Reply:
OH WAIT - that is stat = "identity" in the landmass example... Let me add the explanation for `position = "identity" ` here.

Reply:
And we kind of need it in the one plot where we try to map the colours and fail - because otherwise, the shape changes and that is confusing.